kahuna/public/js/components/gr-panel/gr-panel.js

```diff
@@ -43,6 +229,17 @@ grPanel.controller('GrPanel', [
         ctrl.hasMultipleValues = (val) => Array.isArray(val);
         ctrl.clear = selection.clear;
 
+
+        ctrl.archivedService = archivedService(selection.images$);
+        ctrl.archivedService.onUpdate(updates => selection.updateImages(updates));
+
+        ctrl.metadataService = metadataService(selection.images$);
+        ctrl.metadataService.onUpdate(updates => selection.updateImages(updates));
+
+        ctrl.labelsService = labelsService(selection.images$);
+        ctrl.labelsService.onUpdate(updates => selection.updateImages(updates));
```

Comment:
Not really sure that this makes the most sense - the idea @theefer had of merging an edits stream with this one would be good, but it least it's moved away from controlling the stream out of context.